src/box/lua/init.c

```diff
@@ -129,43 +129,43 @@ extern char session_lua[],
 	merger_lua[];
 
 static const char *lua_sources[] = {
```

Comment:
Minor: It's worth to add a comment regarding this not such obvious structure. However, everybody live without it for many years already, so free to ignore.

Reply:
Furthermore, what is the rule for setting either name or `NULL`?

Reply:
Fair point. Added:

```diff
diff --git a/src/box/lua/init.c b/src/box/lua/init.c
index 79c6cce17..5eccd6902 100644
--- a/src/box/lua/init.c
+++ b/src/box/lua/init.c
@@ -128,6 +128,24 @@ extern char session_lua[],
        console_lua[],
        merger_lua[];
 
+/**
+ * List of box's built-in modules written using Lua.
+ *
+ * Each module is defined as a triplet:
+ *
+ * 1. A file name (without the .lua extension).
+ *
+ *    It is used for prepending error messages and filling
+ *    debug.getinfo() information.
+ * 2. A module name for require().
+ *
+ *    NULL means 'do not register as a module, just execute'.
+ *    Such code shouldn't return any value.
+ *
+ *    Typical NULL usage: code that define functions in
+ *    box or box.internal.
+ * 3. A Lua source code of the module.
+ */
 static const char *lua_sources[] = {
        "box/session", NULL, session_lua,
        "box/tuple", NULL, tuple_lua,
```